packages/taquito-contracts-library/test/taquito-contracts-library.spec.ts

```diff
@@ -90,19 +101,36 @@ describe('ContractsLibrary tests', () => {
           script,
           entrypoints,
         },
+    
       });
     } catch (e: any) {
+      expect
+      expect(validateAddress(contractAddress)).toEqual(0)
       expect(e).toBeInstanceOf(InvalidAddressError);
       expect(e.message).toEqual(`Address is invalid: ${contractAddress}`);
       expect(e).toBeInstanceOf(Error);
     }
   });
 
+  it('do not throw an InvalidAddress error if the contract address is valid', () => {
+    const contractAddress = 'KT1NGV6nvvedwwjMjCsWY6Vfm6p1q5sMMLDY';
+    const contractLib = new ContractsLibrary();
+
+    contractLib.addContract({
+      [contractAddress]: {
+        script,
+        entrypoints,
+      },
+    });
+  });
+
   it('throw an InvalidScriptFormatError error if the script format is invalid', () => {
     const contractAddress = 'KT1NGV6nvvedwwjMjCsWY6Vfm6p1q5sMMLDY';
     const contractLib = new ContractsLibrary();
     const script: any = 'invalid';
 
+    !expect(script.code);
```

Comment:
First time I see except used that way. Can you explain what it does?

Reply:
This was from the previous test author.  I believe it means to check that there is no .code section of the script, t be sure it is invalid.  it says "expect the assertion that there is a script.code to be false"

Reply:
Oh ok, as the line is green and I don't see it on master, I thought it was an addition in the current PR